common/db/empty_timestamp.go

```diff
@@ -0,0 +1,27 @@
+package db
+
+// MongoCanAcceptLiteralZeroTimestamp indicates whether the given server
+// version can accept a literal zero timestamp in a query. See SERVER-88750.
+func MongoCanAcceptLiteralZeroTimestamp(version Version) bool {
```

Comment:
Probably also worth referencing this TOOLS ticket.

Reply:
done